docs/source/releases/v1.6.rst

```diff
@@ -74,6 +74,9 @@ Minor changes
    platform-independent and generates the same for Python 2 and 3.
    Corresponding data migration will update hashes automatically. Take it into
    account that it will take time to run if you have large user base.
+ - Compact basket form is no longer displayed for the products with options
```

Comment:
This looks good to me. The release note doesn't mention the new `has_options` property though, which I think is important to include somewhere in the documentation?